guide/foundations/backups.md

```diff
@@ -0,0 +1,265 @@
+
+To protect against such a *catastrophic event* we can create multiple backups, and distribute them to other locations. This could be a second home, other family members or a safe deposit box. 
+
+As the risk of discovery increases with the multiplying backups, best practice at this level would be to maintain the protection against theft on discovery like in level 3 for all the additional backups. This could again be by using a pass-phrase, a duplicate hardware wallet with a PIN, or a mix. 
+
+{% include image.html
+   image = "/assets/images/guide/foundations/backups/multiple-discoveryprotection.png"
+   retina = "/assets/images/guide/foundations/backups/multiple-discoveryprotection@2x.png"
+   alt-text = "Multiple offline backups with discovery protection"
+%}
+
+**Do** 
+- Document your setup well for next-of-kin now that complexity has increased
+
+**Don’t** 
+- Forget to safely store the pass-phrases or PINs
+
+**Suitable for**
+- Large amounts
+- In locations where catastrophic events are common, or likely to cause severe damage
+
+<br>
+
+#### Level 5
+## Multi-continental backups
+The previous level should offer protection against all but very large catastrophic events. If you want to mitigate against even larger events, like countrywide or continental destruction, further distribution of backups would be necessary.
+
+A recommended setup at this level could include offline metal backups with discovery protection stored in safe-deposit boxes on two or more continents. With the pass-phrase being treated with equal care, backed up in an encrypted password manager AND one or more physical locations on the same continents as, but separate from the metal backups.
+
+
+{% include image.html
+   image = "/assets/images/guide/foundations/backups/multi-continental.png"
+   retina = "/assets/images/guide/foundations/backups/multi-continental@2x.png"
+   alt-text = "Multi-continental backups"
+%}
+
+
+**Do** 
+- Document your setup extremely well and in redundant ways.
+
+**Don’t** 
+- Forget to safely store the extra words or PINs in multiple locations
+
+**Suitable for**
+- Very large amounts
+- In situations or times where high risk, or impact of loss is likely
+
+**Alternatives** 
+At this level, one can imagine many alternative solutions that could provide similar levels of protection, including multiple distributed hardware wallets with the same recovery-phrase, or multi-sig setups. For very large amounts, get expert advice from a specialist.
+
+<br>
+
+# Inheritance instructions
+For any backup setup, sufficient information and understandable instructions for recovering the wallet should be available to next-of-kin.  
+
+Even though it might feel uncomfortable (or unnecessary if you are young and single), it is never too early to document your wallets. If you don’t there is a very high chance that none of the funds will be available to next-of-kin. 
```

Comment:
```suggestion
Even though it might feel uncomfortable (or unnecessary if you are young and single), it is never too early to document your wallets. If no one but you had prior access to the private keys, then no one would be able to recover the funds. 
```

Reply:
This seems less clear, but I tweaked the sentence in a different commit.